homeassistant/components/recorder/statistics.py

```diff
@@ -468,3 +481,16 @@ def no_conversion(val: Any, _: Any) -> float | None:
 
     # Filter out the empty lists if some states had 0 results.
     return {metadata[key]["statistic_id"]: val for key, val in result.items() if val}
+
+
+def validate_statistics(hass: HomeAssistant) -> dict[str, list[ValidationIssue]]:
+    """Validate statistics."""
+    platform_validation: dict[str, list[ValidationIssue]] = {}
+    for platform in hass.data[DOMAIN].values():
+        if not hasattr(platform, "validate_statistics"):
+            continue
+        platform_validation = {
+            **platform_validation,
+            **platform.validate_statistics(hass),
+        }
```

Comment:
No need to copy the whole dict.
```suggestion
        platform_validation.update(platform.validate_statistics(hass))
```

Reply:
Fixed in 375babe